runelite-client/src/main/java/net/runelite/client/plugins/cannon/CannonPlugin.java

```diff
@@ -413,8 +295,33 @@ private void removeCounter()
 		counter = null;
 	}
 
-	private static WorldArea buildCannonWorldArea(WorldPoint worldPoint)
+	private WorldPoint getCannonWorldPoint(int var)
```

Comment:
It's rather confusing to overload the getter of `cannonWorldPoint` like this, a different method name would be helpful to disambiguate them. Additionally, the passed value is always `client.getVarpValue(VarPlayer.DWARF_CANNON_LOCATION)` so it would be better to do that call in this method instead of passing the value in.